docs/admin.md

```diff
@@ -0,0 +1,59 @@
+## Admin Manual for Definitely Typed
+
+Welcome! This is a resource for the person who is on call for DefinitelyTyped. The TypeScript team always has someone 
+assigned to DefinitelyTyped duty, where they do a week on-call. You can see the [schedule here](http://aka.ms/DTRotation).
+
```

Comment:
These two paragraphs are a good start, although eventually I'd like to have a bit more about the philosophy &mdash; that this isn't a code review in the usual sense, but an assessment of category and severity, followed by delegating to the people who actually know how to review the code. Since the bot tags the PRs with category and owners, it's mainly an exercise in co-ordinating between humans, and filling in any places the bot currently misses.

Also, something about how the sliding scale of package popularity affects reviews. The maintainer should review changes to popular packages, and very popular packages &mdash; those with a name you recognise, roughly &mdash; should have signoffs from at least two maintainers. Conversely, it's fine to overlook problems with changes to unpopular packages; in many cases only the author is using them anyway.

Reply:
I agree - I've expanded it!